chamge/common.c

```diff
@@ -24,18 +24,9 @@
 GSettings *
 chamge_common_gsettings_new (const gchar * schema_id)
 {
-  if (glib_check_version (2, 60, 5) && getppid () == 1) {
-    /* Workaround GLib bug where keyfile GSettings backend wasn't registered. */
-    g_autoptr (GSettingsBackend) backend = NULL;
-    g_autofree char *filename = NULL;
+  g_autoptr (GSettingsBackend) backend = NULL;
 
-    filename = g_build_filename (g_get_user_config_dir (), "glib-2.0",
-        "settings", "keyfile", NULL);
+  backend = g_keyfile_settings_backend_new ("/etc/hwangsaeul.conf", "/", NULL);
```

Comment:
The path should be from *mesonbuild* as *sysconfdir*, but for now, it's okay. I'll file an issue to fix later.